data/templates/plugins/openshift-cluster-upgrade.yaml

```diff
@@ -1,53 +1,82 @@
+---
 podSpec:
   restartPolicy: Never
   serviceAccountName: sonobuoy-serviceaccount
+  priorityClassName: system-node-critical
```

Comment:
do we really need this? it's not showing good results when using the PC, removing  the tolerations/dedicated node. The environment (plugins) are eventually evicted.

Reply:
fixed